src/index.ts

```diff
@@ -164,6 +170,29 @@ function sendTransactions(txs: Transaction[], requestId?: RequestId): SentSDKMes
   return message;
 }
 
+/**
+ * Request Safe app to send transactions
+ * @param txs
+ */
+function sendTransactionsWithParams(
```

Comment:
What do you think about changing the input parameters to an object to avoid writing `undefined` as the value of `params` when `params` is `undefined` but `requestId` is not?

Reply:
Let's do it